core/src/main/java/com/sequenceiq/distrox/v1/distrox/converter/DistroXV1RequestToStackV4RequestConverter.java

```diff
@@ -112,6 +117,10 @@ public StackV4Request convert(DistroXV1Request source) {
         request.setExternalDatabase(getIfNotNull(source.getExternalDatabase(), databaseRequestConverter::convert));
         request.setEnableLoadBalancer(source.isEnableLoadBalancer());
         request.setVariant(source.getVariant());
+        if (CloudPlatform.AWS.equals(source.getCloudPlatform()) &&
+                entitlementService.awsNativeSingleAzDatahubEnabled(ThreadBasedUserCrnProvider.getAccountId())) {
```

Comment:
should we say **or multiazentitlementGranted()**

Reply:
how multi AZ comes into picture here? here we need to enable native even if multiAz not enabled